tests/infrastructure/workload_availability/remediation_fencing/test_nodehealthcheck_fenceagent_remediation.py

```diff
@@ -21,13 +21,7 @@
     wait_for_nodehealthcheck_enabled_phase,
 )
 
-pytestmark = [
-    pytest.mark.usefixtures(
-        "skip_if_workers_vms",
-        "skip_on_aws_cluster",
-    ),
-    pytest.mark.node_remediation,
-]
+pytestmark = pytest.mark.node_remediation_ipmi_enabled
```

Comment:
Does this require fully capable cluster?

Reply:
it will need ipmi and that's possible in rdu clusters and ibm.We don't need full capable cluster for it

Reply:
How do you ensure that this test is only run against ibm or rdu?

Reply:
I have a JIRA CNV-48126 to track creation of nhc lanes and it needs ipmi enabled cluster only. I have it as requirement .

Reply:
Currently, this would try to run against a PSI cluster - since there is nothing indicating it requires special_infra/fully capable cluster. This would impact other lanes scheduled for the same cluster. Please watch the recorded programmatic skip meeting and let me/ Devops know if you have any questions.